bin/xml-2-json.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Will convert a old style xml feed to json.
+# vespa must be running and the searchdefinition deployed.
+
+base=$(basename $1 ".xml")
+wrap-xml-feed.sh $1 | vespa-feed-perf -o "$base.json"
```

Comment:
please ensure that $base.json ends with a newline if not completely empty.

Reply:
Fixed